README.md

```diff
@@ -1,6 +1,7 @@
 ![Are you mocking me?](https://fakeiteasy.github.io/img/fakeiteasy_logo_256.png)
 
-[![NuGet version](https://img.shields.io/nuget/v/FakeItEasy.Analyzers.svg?style=flat)](https://www.nuget.org/packages/FakeItEasy)
+[![CSharp Analyzer NuGet version](https://img.shields.io/nuget/v/FakeItEasy.Analyzer.CSharp.svg?style=flat&label=C%23)](https://www.nuget.org/packages/FakeItEasy.Analyzer.CSharp)
+[![Visual Basic Analyzer NuGet version](https://img.shields.io/nuget/v/FakeItEasy.Analyzer.VisualBasic.svg?style=flat&label=Visual%20Basic)](https://www.nuget.org/packages/FakeItEasy.Analyzer.VisualBasic)

```

Comment:
I think the label should still say "nuget", otherwise it's not obvious what the badge represent. If we want to specify the language, we can write "nuget (C#)".

*Or* we can include the NuGet icon (`logo=nuget`). It looks like this:
![image](https://user-images.githubusercontent.com/1864281/70919749-b4599200-2021-11ea-8539-54ea7642dc05.png)

Reply:
Changing logo as you suggest, and keeping existing labels. Nice idea.